src/libraries/xchaininit/src/xchain_command.cc

```diff
@@ -750,6 +750,17 @@ int parse_execute_command(const char * config_file_extra, int argc, char * argv[
         "account_addr", queryBlock_account, "Account address to query its block block information. if you do not add this parameters, your default account will be queried.");
     queryBlock_app->callback(std::bind(&ApiMethod::query_block, &topcl.api, std::ref(queryBlock_account), std::ref(queryBlock_height), std::ref(out_str)));
 
+    // query blocks by height
```

Comment:
for topio, the naming style is using `query` instead of `get`, for example, queryBlock, queryAccount